README.md

```diff
@@ -45,6 +46,8 @@ $ python <(curl https://raw.githubusercontent.com/screwdriver-cd/screwdriver/mas
 
 Screwdriver already [defaults most configuration](config/default.yaml), but you can override defaults using a `local.yaml` or environment variables.
 
+For more information, see the [cluster management docs](https://github.com/screwdriver-cd/guide/blob/master/docs/cluster-management/configure-api.md#managing-the-api).
```

Comment:
I think we should clarify here that cluster management is necessary for most users. Otherwise, if you had read this when setting things up, you might have still glossed over the docs thinking that it was irrelevant. At minimum, we could say something like "To continue set up, follow the instructions for cluster management".